updateParticipantData.json

```diff
@@ -407,5 +407,110 @@
 	"173836415.266600170.110349197": {
 		"dataType": "array",
 		"mustExist": false
```

Comment:
Note: spacing looks accurate in VS Code
<img width="388" alt="Screenshot 2023-12-06 at 6 11 22 PM" src="https://github.com/episphere/connectFaas/assets/93854858/b90b8a2b-4118-4bd1-bf4c-da078e25895d">